frontend/src/index.js

```diff
@@ -8,7 +8,13 @@ import { Auth0Provider } from "@auth0/auth0-react";
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <Auth0Provider>
+    <Auth0Provider
+    domain="dev-3a82s6v3xowkao55.us.auth0.com"
```

Comment:
Add `domain`, `clientId` both in `.env` file, both details are sensitive they should not be publicly available.

Reply:
I will change it in the next pr